doc/SetupRClone.md

```diff
@@ -60,66 +61,74 @@ These instructions will speed you through the process with good defaults. If you
+
+   It has been confirmed that this process works with Google Drive using these [instructions](https://rclone.org/drive/). If you are using another storage system, please feel encouraged to create an "Issue" describing your challenges and/or your success.
 
-    If you are using Google Drive it is important to set the correct [scope](https://rclone.org/drive/#scopes). Carefully read the documentation on [scopes on rclone](https://rclone.org/drive/#scopes) as well as [Google Drive](https://developers.google.com/drive/api/v3/about-auth). The `drive.file` scope is recommended.
+   If you are using Google Drive it is important to set the correct [scope](https://rclone.org/drive/#scopes). Carefully read the documentation on [scopes on rclone](https://rclone.org/drive/#scopes) as well as [Google Drive](https://developers.google.com/drive/api/v3/about-auth). The `drive.file` scope is recommended.
 
-    **Important:** During the `rclone config` process you will specify a name for the configuration. The rest of the document will assume the use of the name `gdrive`; replace this with your chosen configuration name.
+   **Important:** During the `rclone config` process you will specify a name for the configuration. The rest of the document will assume the use of the name `gdrive`; replace this with your chosen configuration name.
 
 1. Run this command:
    ```
    export RCLONE_DRIVE="gdrive"
    ```
+
 # Step 3: Verify and create storage directory
 
 1. Run the following command to see the name of the remote drive you just created.
-    ```
-    rclone listremotes
-    ```
-    If you don't see the name there, something went wrong. Go back through the `rclone config` process.
+   ```
+   rclone listremotes
+   ```
+   If you don't see the name there, something went wrong. Go back through the `rclone config` process.
 1. Run this command:
-    ```
-    rclone lsd "$RCLONE_DRIVE":
-    ```
-    You should not see any files listed. If you do then you did not set your scope correctly during the `rclone config` process.
+   ```
+   rclone lsd "$RCLONE_DRIVE":
+   ```
+   You should not see any files listed. If you do then you did not set your scope correctly during the `rclone config` process.
 1. Choose the name of a folder to hold the archived clips. These instructions will assume you chose the name `TeslaCam`. Substitute the name you chose for this name. Run this command:
-    ```
-    export RCLONE_PATH="TeslaCam"
-    ```
+   ```
+   export RCLONE_PATH="TeslaCam"
+   ```
 1. Run the following command to create a folder which will hold the archived clips.
-    ```
-    rclone mkdir "$RCLONE_DRIVE:TeslaCam"
-    ```
+   ```
+   rclone mkdir "$RCLONE_DRIVE:TeslaCam"
+   ```
 1. Run this command again:
-    ```
-    rclone lsd "$RCLONE_DRIVE":
-    ```
-Confirm that the directory `TeslaCam` is present. If not, start over.
+   `    rclone lsd "$RCLONE_DRIVE":`
```

Comment:
This also looks wrong to me, not sure why Prettier would do this: in the "Prettier" code, the command is no longer on a line by itself, and has 4 spaces prefixed to it.

Reply:
Also some craziness from the existing content, fixed.